project/src/components/card/card.tsx

```diff
@@ -1,19 +1,29 @@
 import { Link } from 'react-router-dom';
+import { Offer } from '../../types/offer';
 
-function Card(): JSX.Element {
+type CardProps = {
+  cardOffer: Offer;
+}
+
+function Card({cardOffer}: CardProps): JSX.Element {
+
+  const {previewImage, isPremium, price, title, type, isFavorite, rating, id} = cardOffer;
 
   return (
-    <article className="cities__place-card place-card">
+    <>
+      {isPremium
+        ?
+        <div className="place-card__mark">
+          <span>Premium</span>
+        </div>
+        : null}
```

Comment:
```suggestion
      {isPremium && (
        <div className="place-card__mark">
          <span>Premium</span>
        </div>
      )}
```

в реакте принято пользоваться ленивыми вычислениями и тот код, что переносится, можно в круглые скобки взять.